manifests/cf-manifest/cloud-config/030-logsearch.yml

```diff
@@ -21,13 +21,13 @@ vm_types:
     network: (( grab vm_types.medium.network ))
     env: (( grab meta.default_env ))
     cloud_properties:
-      instance_type: (( grab vm_types.medium.cloud_properties.instance_type ))
+      instance_type: (( grab vm_types.large.cloud_properties.instance_type ))
```

Comment:
I think it would be simpler to put m3.large here instead of defining a new instance type, just so that you can grab a single property from it...